spec/fixtures/pages/video.html

```diff
@@ -0,0 +1,14 @@
+<html>
+<body>
+<video id="videoPlayer" src="/video.mp4" autoplay muted></video>
```

Comment:
It looks like the mp4 in this PR is 1MB? That's a little on the larger side to commit to the repo forever :/

Reply:
Yeah any thoughts on what to do there? We need a larger video for the test to work. I guess we could try to come up with a script to generate the video on test-run?

Reply:
you mentioned [here](https://github.com/electron/electron/pull/22955#issuecomment-622066127) that you found a 63kb video that triggers the issue. That seems sufficiently small. If that's not the case, then yeah a script to generate a video seems reasonable.

Would it trigger the issue to have the custom protocol handler stream back a very small amount of the video in the first chunk? e.g. just a few KB. Then wait a bit and return the rest.

Reply:
@nornagon sorry that comment was unclear, the the smaller video doesn't trigger the issue.

I'll try your second suggestion first. Generating an mp4 would be a bit harder.

Reply:
I was able to find a relatively small webm generator called Whammy and used that.